### PR TITLE
Update CHANGELOG for v2.10.0 release

### PR DESCRIPTION
**Pull request by JLXIA**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Release 2.10.0 18-05-2020
+
+- Implement `denied_consumer_blocked` and `denied_consumer_error` stats (#156)
+- Refactor callback for testability and implement `denied_control_plane_fault` (#155)
+- Add statistical counter for check/allocateQuota/report call status (#151)
+- Fix network fail open for Unavailable CheckResponse errors (#152)
+- Add latency in ServiceControl statistics (#146)
+- Destruct invalid json keys in grpc auth lib (#144)
```
Comment (nareddyt) [inline CHANGELOG.md:8]: You can remove this

Comment (TAOXUY) [inline CHANGELOG.md:8]: remove this

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# Release 2.10.0 18-05-2020
+
+- Implement `denied_consumer_blocked` and `denied_consumer_error` stats (#156)
+- Refactor callback for testability and implement `denied_control_plane_fault` (#155)
```
Comment (TAOXUY) [inline CHANGELOG.md:4]: Only keep this

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+- Add statistical counter for check/allocateQuota/report call status (#151)
+- Fix network fail open for Unavailable CheckResponse errors (#152)
```
Comment (TAOXUY) [inline CHANGELOG.md:6]: remove this

File: CHANGELOG.md (hunk 2 above)
Comment (TAOXUY) [reply CHANGELOG.md:4]: implement `denied_control_plane_fault`

**Pull request merged**